samples/multiplatform/settings.gradle.kts

```diff
@@ -0,0 +1,15 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
```

Comment:
why do we need google()?

Reply:
just leftover from tying to add android as a platform